apps/editing-toolkit/editing-toolkit-plugin/premium-content/premium-content.php

```diff
@@ -237,9 +300,66 @@ function premium_content_container_render( $attributes, $content ) {
 		return '';
 	}
 
+	// Show upgrade nudge.
+	if (
+		! premium_content_required_plan_checks() &&
+		premium_content_current_user_can_edit()
+	) {
+		$upgrade_nudge = premium_content_render_upgrade_nudge();
+		return $upgrade_nudge . $content;
+	}
+
+	// Stripe connection nudge.
+	if (
+		! premium_content_membership_checks() &&
+		premium_content_current_user_can_edit()
+	) {
+		$stripe_nudge = premium_content_render_stripe_nudge();
+		return $stripe_nudge . $content;
+	}
+
 	return $content;
 }
 
+/**
+ * Server-side rendering for the upgrade nudge.
+ *
+ * @return string Final content to render.
+ */
+function premium_content_render_upgrade_nudge() {
+	$required_plan = ( defined( 'IS_WPCOM' ) && IS_WPCOM ) ? 'personal-bundle' : 'jetpack_personal';
+
+	if ( function_exists( 'jetpack_require_lib' ) ) {
+		jetpack_require_lib( 'components' );
+		return Jetpack_Components::render_upgrade_nudge(
```

Comment:
Need a \ here before Jetpack_Component.

Reply:
I think along with the `jetpack_require_lib` check, there should also be a `class_exists` check on the `Jetpack_Components` class. It's still possible even if the library loading code is present for some reason Jetpack_Components is not.